getting-started/developers.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Developers
+description: Get started with Coder as a developer.
+---
+
+This article will walk you through the process of getting started with a Coder
+workspace and a web development project featuring Node.js and React.js. You'll
+learn how to:
+
+- Connect and authenticate to your Git provider;
+- Create a workspace and add the files for a sample project that's available via
+  GitHub;
+- Create a dev URL and use it to preview changes to your project.
```

Comment:
These 3 points don't clearly align to the 4 steps below. I'm thinking the following can help clarify this:

## Step 1: Connect Coder to your Git provider

## Step 2: Create a workspace

## Step 3: Add a sample project to your workspace

## Step 4: Create a dev URL and preview changes to your project

Thoughts?